FluentUI.Demo/src/main/res/layout/activity_persistent_bottom_sheet.xml

```diff
@@ -40,6 +40,13 @@
             android:layout_marginTop="@dimen/default_layout_margin"
             android:text="@string/toggle_sheet_content" />
 
```

Comment:
Please check the reply to conversation, regarding having a separate PR for these changes related to persistent bottomsheet

Reply:
Replied to the previous conversation